test/ci-cd/terraform/cloudbuild/cloudbuild_trigger.tf

```diff
@@ -219,3 +219,54 @@ resource "google_cloudbuild_trigger" "uc_mftp_model_eval_build" {
     }
   }
 }
+
+resource "google_cloudbuild_trigger" "uc_federated_learning_terraform" {
+  filename = "test/ci-cd/cloudbuild/uc-federated-learning-terraform.yaml"
+  ignored_files = [
+    "platforms/gke/base/core/README.md",
+    "platforms/gke/base/core/use-cases/README.md",
```

Comment:
This should probably be `platforms/gke/base/use-cases/federated-learning/README.md`

Reply:
Added both because there's `platforms/gke/base/core/use-cases/README.md` as well

Reply:
That file wouldn't get picked up because it is not in any of the `included_files`.

Reply:
oh right, removing.